docs/develop/dotnet/failure-detection.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: failure-detection
+title: Failure Detection .NET SDK feature guide
+sidebar_label: Failure Detection
+description: The Failure Detection section of the Temporal Developer's guide covers the many ways to detect failure with your Temporal application.
+toc_max_heading_level: 4
+keywords:
+  - guide context
+  - how to
+  - dotnet
+  - sdk
+  - failure detection
+tags:
+  - guide-context
+  - how-to
+  - dotnet
+  - sdk
+  - failure-detection
+---
+
+## Failure Detection {#failure-detection}
+
+## Workflow timeouts {#workflow-timeouts}
```

Comment:
The title of this document is failure _detection_ but this is about the presence of timeout failures, not about detecting them